DependencyInjection/CmfSeoExtension.php

```diff
@@ -208,26 +200,22 @@ private function loadAlternateLocaleProvider($config, ContainerBuilder $containe
             : $config['provider_id'];
 
         if (!$alternateLocaleProvider) {
-            return;
+            throw new InvalidConfigurationException('Alternate locale provider enabled but none defined. You need to enable PHPCR or configure alternate_locale.provider_id');
```

Comment:
this might be a BC break, but i think it would be a weird situation if this happens.